recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
         - def_snprintf.patch  # [win]
 
 build:
-    number: 3
+    number: 1
```

Comment:
Unfortunately no. This exactly what happened with `cloog` we had 10, but it chose 0. There may be a solver bug involved. Not sure. I need to generate a test case for @mcg1969 so we can discuss. Though channel preference will help this immensely if not solve it completely.